### DISCUSSION TITLE
Australia/New Zealand now have SBAS SouthPAN - not yet implemented in iNav 6.1.1

### DISCUSSION THREAD

**Discussion by JetsRfaster** [iNavFlight/inav#9313] [General]
ref https://www.ga.gov.au/__data/assets/pdf_file/0011/123320/SBAS-STN-0001_02_SDD-OS.pdf
     https://www.amsa.gov.au/safety-navigation/navigation-systems/satellite-based-augmentation-system-sbas

Can we add SouthPAN SBAS for users in this Australia and New Zealand region please?  
Currently, if SBAS is set to **auto** in 6.1.1 the GPS struggles to get a position.  
If disabled we get a much lower position accuracy and longer sat acquisition times.

thanks

**Comment by stronnag** [comment]
See #9315 .
